src/charm.py

```diff
@@ -446,7 +446,8 @@ def _reload_configuration(self) -> None:
         if self._is_config_valid() and container.can_connect():
             layer_config = self._create_layer_config()
             container.add_layer(SERVICE_NAME, layer_config, combine=True)
-            container.pebble.replan_services()
+            container.stop(SERVICE_NAME)
```

Comment:
Can you add a comment explaining the change and a link to the opened issue, if any?

Reply:
Done :)